src/sonic-config-engine/sonic-cfggen

```diff
@@ -57,7 +57,9 @@ def sort_by_port_index(value):
     if not value:
         return
     if isinstance(value, list):
-        value.sort(key = lambda k: int(k[8:]))
+        value.sort(
```

Comment:
Why do we need this change ?

Reply:
It's due to internal interfaces names, they start with "Ethernet-BPxxx". Without this change sorting fails. Adding 1024 so that internal interfaces are sorted to the end, no particular reason.